Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -221,20 +229,30 @@ public struct PackageBuilder {
 
         switch viableRoots.count {
         case 0:
-            return packagePath
+            return .root
         case 1:
-            return packagePath.appending(component: viableRoots[0])
+            return AbsolutePath.root.appending(component: viableRoots[0])
         default:
             // eg. there is a `Sources' AND a `src'
             throw ModuleError.invalidLayout(.multipleSourceRoots(viableRoots.map{ packagePath.appending(component: $0).asString }))
         }
     }
 
+    /// Converts a rerouted path to the real path.
+    // FIXME: This is needed currently because modules expect actual paths of the sources.
```

Comment:
Yeah, that is what I think should happen.

I want `Package`s to be agnostic to where they actually are on disk.